tests/README.md

```diff
@@ -0,0 +1,93 @@
+# E2E Tests
```

Comment:
Currently I named this package as `tests` but I like the singular form more. That's why once we delete current `test` directory, I will rename it again to `test`. For now I wanted to distinguish the difference between the packages and didn't want to mix the files.

But of course I'm also open for different suggestions how to approach this 🙂